.github/workflows/ci-go.yml

```diff
@@ -27,12 +27,12 @@ jobs:
           go-version: 1.18.0
         id: go
 
-      - uses: pnpm/action-setup@v2
+      - uses: pnpm/action-setup@v2.2.2
         with:
-          version: 6.32.11
+          version: 7.2.1
 
       - name: Setup Node.js environment
-        uses: actions/setup-node@v3
+        uses: actions/setup-node@v2
```

Comment:
Is there a reason to downgrade this?

Reply:
I am copying the recipe from pnpm.io https://pnpm.io/continuous-integration#github-actions

Reply:
That error has been very frequent over the past two days. I've been trying random things to address.